distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -1056,7 +1126,7 @@ list_descendants file =
    construct the path from text - we instead find its root (or base directory if
    relative) and rely on the `/` operator to resolve all the parts, allowing the
    data link logic to kick in.
-resolve_local_file (path : Text) = handle_invalid_path path <|
+private resolve_local_file (path : Text) = handle_invalid_path path <|
```

Comment:
- Making this method `private` seems fine.
- can we make more methods `private`?
   - ideally all that have _private comment_
   - like `list_immediate_children`